multiplayer_server/rooms/Game.php

```diff
@@ -430,6 +430,7 @@ public function finish_race($player)
 
         if ($player->finished_race === false && !isset($player->race_stats->finish_time) && $player->race_stats->drawing === false && $this->begun === true) {
             $finish_time = microtime(true) - $this->start_time;
+            if ($finish_time > 31536000) $finish_time = 0; // if the race time is >1 year, set it to 0. This is just a glitch protection.
```

Comment:
You could probably even make this a day. I don't think there's ever been a level that takes a day to complete.

Reply:
Nope. There are levels that take days and even months to complete.

Reply:
The current UNIX timestamp is 1522 million and counting; 31.5 million is good enough to catch those botters. I set it stupidly large on purpose.

Reply:
@MoonWithSunshine so a year is OK? Good.